### PR TITLE
Jb app refact

### PR DESCRIPTION
**Pull request by Nightingale1997**

### CODE REVIEW

File: src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java [hunk 1]
```diff
@@ -5,117 +5,280 @@
...
+            }
+
+            @Override
+            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
+                transformPWM();
+            }
+        });
+
+        /**
+         * Listener for bottomBar
```
Comment (isZumpo) [inline src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java]: A better naming convention for top and bottom bars would be great. To further explain which one is what.

File: src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java [hunk 2]
```diff
@@ -5,117 +5,280 @@
...
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        topBar = (SeekBar) findViewById(R.id.TopBar);
+        bottomBar = (SeekBar) findViewById(R.id.BottomBar);
+
+        /*Listener for change in SeekBars (Sliders) */
+
+        /**
+     * Listener for topBar
```
Comment (isZumpo) [inline src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java]: A better naming convention for top and bottom bars would be great. To further explain which one is what.

File: src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java [hunk 3]
```diff
@@ -5,117 +5,280 @@
...
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Send a message through the socket.
+     *
+     * @param message the message that will be sent to the MOPED
+     */
+    public static void send(Object message) {
```
Comment (isZumpo) [inline src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java]: Could have a more describing name like sendSocket or something.

**Review by isZumpo — CHANGES_REQUESTED** [review]
It looks good. However javadocs formating could be more consistent. Sometimes it's one row over a method while at other times two rows.

File: src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java (hunk 2 above)
Comment (konglobemeralt) [reply src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java]: Fixat, steeringBar och speedBar istället nu

File: src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java (hunk 3 above)
Comment (konglobemeralt) [reply src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java]: Dock skickas väl bara ett meddelande via socketen? Men kanske sendToMoped?

Comment (isZumpo) [reply src/app/WirelessIno/app/src/main/java/se/sics/sse/fresta/wirelessino/Main.java]: True! I'm ok with that

**Comment by konglobemeralt** [comment]
The bar names now better reflect the functionality. And all javadocs are one line above functions.

**Review by isZumpo — APPROVED** [review]
All good even though I like to have javadocs comments directly above the methods/classes as it's easier to read in github.

**Pull request merged**